### PR TITLE
Translate Ruby 2.4.0preview2 (es)

### PR DESCRIPTION
**Pull request by FerPerales**

### CODE REVIEW

File: es/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+---
+
+Nos complace anunciar el lanzamiento de Ruby 2.4.0-preview2.
+
+Ruby 2.4.0-preview2 es la segunda muestra previa de Ruby 2.4.0.
+Este preview2 es liberado para obtener retroalimentación de la comunidad.
+Siéntete libre de
+[enviar tus comentarios](https://bugs.ruby-lang.org/projects/ruby/wiki/HowToReport)
+ya que aún puedes influír en las funcionalidades.
+
```
Comment (gfvcastro) [inline es/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md]: Remove line break.

File: es/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+## [Unificación de Fixnum y Bignum en Integer](https://bugs.ruby-lang.org/issues/12005)
+
+A pesar de que [ISO/IEC 30170:2012](http://www.iso.org/iso/iso_catalogue/catalogue_tc/catalogue_detail.htm?csnumber=59579)
+no especifica detalles sobre la clase Integer,
+Ruby tenía dos clases visibles para enteros: Fixnum y Bignum.
+Ruby 2.4 las unifica dentro de Integer.
+Todas las extensiones de C que tocan las clases Fixnum o Bignum necesitan ser reparadas.
+
+Ve también [el incidente](https://bugs.ruby-lang.org/issues/12005) y las [diapositivas de ark](http://www.a-k-r.org/pub/2016-09-08-rubykaigi-unified-integer.pdf).
+
```
Comment (gfvcastro) [inline es/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md]: Remove line break.

File: es/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+title: "Ruby 2.4.0-preview2 Released"
+author: "naruse"
+translator: "Fernando Perales"
+date: 2016-09-08 18:00:00 +0900
+lang: es
+---
+
+Nos complace anunciar el lanzamiento de Ruby 2.4.0-preview2.
+
+Ruby 2.4.0-preview2 es la segunda muestra previa de Ruby 2.4.0.
```
Comment (gfvcastro) [inline es/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md]: What do you think about update this `muestra previa` to `vista previa`?

File: es/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md (hunk 1 above)
Comment (gfvcastro) [inline es/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md]: Remove line break.

**Comment by FerPerales** [comment]
@gfvcastro Thanks for the ready. I've attended all suggestions 🙇

**Pull request merged**